src/operator/bufferCount.ts

```diff
@@ -19,7 +19,11 @@ import {Observable} from '../Observable';
  * @returns {Observable<T[]>} an Observable of arrays of buffered values.
  */
 export function bufferCount<T>(bufferSize: number, startBufferEvery: number = null): Observable<T[]> {
-  return this.lift(new BufferCountOperator(bufferSize, startBufferEvery));
+  return this.lift(new BufferCountOperator<T>(bufferSize, startBufferEvery));
+}
+
+export interface BufferCountSignature<T> {
```

Comment:
Compare to previous type definitions having n-number of param (combinelatest, etcs) these kind of operators have single signature only. Is it necessarily required to export signature then import in coreoperators?